nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/pom.xml

```diff
@@ -124,4 +124,92 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
+    <profiles>
+        <profile>
+            <!-- Custom profile for Groovy tests -->
+            <id>groovyUnitTest</id>
```

Comment:
No, this is an artifact because I was using Groovy tests internally before [NIFI-1365] was accepted. Will remove.